cypress_test/integration_tests/desktop/writer/file_properties_spec.js

```diff
@@ -40,7 +40,9 @@ describe(['tagdesktop', 'tagnextcloud', 'tagproxy'], 'File Property Tests', func
 
 		// Add property
 		cy.cGet('#add.ui-pushbutton').click();
-		cy.cGet('#namebox-input-dialog').type('Mailstop');
+		cy.cGet('#namebox-input-dialog')
+			.type('Mailstop')
+			.should('have.value', 'Mailstop');
```

Comment:
there is no sense to check it here (we just typed it).
Please move below `// Check property saved`
we reopen the dialog and check if it was really saved

Reply:
Oh, I misunderstood what was wrong with your last comment.